core/src/test/java/org/springframework/security/core/authority/mapping/SimpleAuthoritiesMapperTests.java

```diff
@@ -25,18 +25,19 @@
 import org.springframework.security.core.authority.AuthorityUtils;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
 
 /**
  * @author Luke Taylor
  */
 public class SimpleAuthoritiesMapperTests {
 
-	@Test(expected = IllegalArgumentException.class)
+	@Test
 	public void rejectsInvalidCaseConversionFlags() throws Exception {
 		SimpleAuthorityMapper mapper = new SimpleAuthorityMapper();
 		mapper.setConvertToLowerCase(true);
 		mapper.setConvertToUpperCase(true);
-		mapper.afterPropertiesSet();
+		assertThatIllegalArgumentException().isThrownBy(mapper::afterPropertiesSet);
```

Comment:
Since this method no longer throws an Exception, should that be cleaned up? There may be other methods to which this clean up might also need applying.